### DISCUSSION TITLE
help me add the limetorrents plugin to qBittorrents

### DISCUSSION THREAD

**Discussion by Lauremideus** [qbittorrent/search-plugins#208] [Q&A]
Hi! I can't install the official limetorrents search plugin in Mac qBittorrent v4.4.3 (64-bit).

https://github.com/qbittorrent/search-plugins/blob/master/nova3/engines/limetorrents.py

I get the message: Couldn't install "limetorrents" search engine plugin. Plugin is not supported.

Please help me add the limetorrents plugin to qBittorrents. Thank you so much!

**Comment by Scare1** [comment]
I'm just working on a better one for LimeTorrents.to, might be ready later today. If you want, you can try it out as I have it now: https://scare.ca/dl/qBittorrent/limetorrents.py

**Reply by Lauremideus** [reply]
Thank you for your prompt reply and for sharing your plugin! It's not working for me. 
![1](https://user-images.githubusercontent.com/113245075/189566662-fa2dda88-7ef6-430f-9b64-89dd6765678b.png)

**Comment by Scare1** [comment]
Hmm, yeah, it's not quite right yet, sorry. I did mention it was still in development. :)

If you were searching for the movie, you probably have to enter "Spider-Man" (its official name, https://www.imdb.com/title/tt10872600/ ), or as 2 separate words. Looks like the trackers have it both with the "-" and without, and as 1 word or 2.

Just now I installed the official LimeTorrents plugin using this URL: https://raw.githubusercontent.com/qbittorrent/search-plugins/master/nova3/engines/limetorrents.py
and it installed fine. Had to uninstall my (newer) plugin first of course.

**Comment by Scare1** [comment]
Small difference, my version uses https://www.limetorrents.to and the official one uses https://limetor.com which redirects to https://www.limetorrents.lol/ .
